### ISSUE TITLE
ADALError: BROKER_AUTHENTICATOR_NOT_RESPONDING on Unihertz Titan

### ISSUE THREAD

**Issue by malwaremagpie** [AzureAD/azure-activedirectory-library-for-android#1491]
* Expected behavior
Users can use the Authenticator app for password-less phone sign-in.

* Actual behavior
Error generated
BROKER_AUTHENTICATOR_NOT_RESPONDING:Failed to bind the service in broker app ver:1.16.3 Android 28
Appears related to debug message (i.e. not an error?) OpenSSLLib: OpensslErr with function ASN1_get_object in file: external/boringssl/src/crypto/asn1/asn1_lib.c 

* Steps to reproduce the problem
1) Install Company Portal
2) InTune device
3) Install Authenticator app
4) Attempt to register device / enable phone sign-in

* Specifications like the version of the library, operating system etc.
Unihertz Titan phone running Android 9, kernel 4.4.146 #28 Wed Jan 15 19:13:07 CST 2020, build number Titan_EEA_20191205
Wireless update version Titan_EEA_20191205_20200115-1427

* Relevant logs and traces

May be related to #1364 and #1107 

From Company Portal app
02-11 13:02:16.815 27887 28135 D OpenSSLLib: OpensslErr:Module:12(177:); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:168;Function:ASN1_get_object
02-11 13:02:16.815 27887 27991 I EventSender: {"ipv":false,"pvm":null,"rej":0,"bln":0,"acc":1,"efi":[]}
02-11 13:02:16.815 27887 27991 I FileStorage: Discarding file
02-11 13:02:16.815 27887 28135 D OpenSSLLib: OpensslErr:Module:12(177:); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:168;Function:ASN1_get_object
02-11 13:02:16.816 27887 27991 I FileStorage: Read 1 events from file
02-11 13:02:16.820 27887 27991 I e.companyporta: The ClassLoaderContext is a special shared library.
02-11 13:02:16.821 27887 27991 I System.out: [OkHttp] sendRequest<<
02-11 13:02:16.926 27887 27991 I EventSender: {"ipv":false,"pvm":null,"rej":0,"bln":0,"acc":1,"efi":[]}
02-11 13:02:16.926 27887 27991 I FileStorage: Discarding file
02-11 13:02:16.927 27887 27991 I FileStorage: Read 1 events from file
02-11 13:02:16.935 27887 27991 I e.companyporta: The ClassLoaderContext is a special shared library.
02-11 13:02:16.937 27887 27991 I System.out: [OkHttp] sendRequest<<
02-11 13:02:17.049 27887 27991 I EventSender: {"ipv":false,"pvm":null,"rej":0,"bln":0,"acc":1,"efi":[]}
02-11 13:02:17.050 27887 27991 I FileStorage: Discarding file
02-11 13:02:17.051 27887 27991 I FileStorage: Read 1 events from file
02-11 13:02:17.059 27887 27991 I e.companyporta: The ClassLoaderContext is a special shared library.
02-11 13:02:17.060 27887 27991 I System.out: [OkHttp] sendRequest<<
02-11 13:02:17.087 27887 28012 I EventHandler: Draining All events
02-11 13:02:17.087 27887 28012 I NormalEventHandler: Writing 1 events to disk
02-11 13:02:17.088 27887 28012 I AbstractHandler: File 9016d03c-3ce1-4ef7-b0d9-9a092b241ae9.norm.cllevent is in use still
02-11 13:02:17.089 27887 28012 I FileStorage: Creating new file
02-11 13:02:17.089 27887 28012 I AbstractHandler: File 825dbb13-a008-4b54-b508-3456e37a1bde.crit.cllevent is in use still
02-11 13:02:17.090 27887 27992 I EventQueueWriter: Starting upload
02-11 13:02:17.090 27887 27992 I EventQueueWriter: Skipping send, event sending is already in progress on different thread.
02-11 13:02:17.144 27887 27991 I EventSender: {"ipv":false,"pvm":null,"rej":0,"bln":0,"acc":1,"efi":[]}
02-11 13:02:17.144 27887 27991 I FileStorage: Discarding file
02-11 13:02:17.145 27887 27991 I FileStorage: Read 1 events from file
02-11 13:02:17.152 27887 27991 I e.companyporta: The ClassLoaderContext is a special shared library.
02-11 13:02:17.154 27887 27991 I System.out: [OkHttp] sendRequest<<
02-11 13:02:17.252 27887 27991 I EventSender: {"ipv":false,"pvm":null,"rej":0,"bln":0,"acc":1,"efi":[]}
02-11 13:02:17.252 27887 27991 I FileStorage: Discarding file
02-11 13:02:17.254 27887 27991 I FileStorage: Read 1 events from file
02-11 13:02:17.261 27887 27991 I e.companyporta: The ClassLoaderContext is a special shared library.
02-11 13:02:17.264 27887 27991 I System.out: [OkHttp] sendRequest<<
02-11 13:02:17.362 27887 27991 I EventSender: {"ipv":false,"pvm":null,"rej":0,"bln":0,"acc":1,"efi":[]}
02-11 13:02:17.362 27887 27991 I FileStorage: Discarding file
02-11 13:02:17.364 27887 27991 I FileStorage: Read 1 events from file
02-11 13:02:17.373 27887 27991 I e.companyporta: The ClassLoaderContext is a special shared library.
02-11 13:02:17.376 27887 27991 I System.out: [OkHttp] sendRequest<<
02-11 13:02:17.583 27887 27991 I EventSender: {"ipv":false,"pvm":null,"rej":0,"bln":0,"acc":1,"efi":[]}
02-11 13:02:17.584 27887 27991 I FileStorage: Discarding file
02-11 13:02:17.585 27887 27991 I FileStorage: Read 1 events from file
02-11 13:02:17.593 27887 27991 I e.companyporta: The ClassLoaderContext is a special shared library.
02-11 13:02:17.595 27887 27991 I System.out: [OkHttp] sendRequest<<
02-11 13:02:17.694 27887 27991 I EventSender: {"ipv":false,"pvm":null,"rej":0,"bln":0,"acc":1,"efi":[]}
02-11 13:02:17.694 27887 27991 I FileStorage: Discarding file
02-11 13:02:17.695 27887 27991 I EventQueueWriter: Sent 120 events.
02-11 13:02:18.138   466  2310 I BufferQueueProducer: [com.microsoft.windowsintune.companyportal/com.microsoft.intune.companyportal.home.presentationcomponent.implementation.HomeActivity#0](this:0x75d07d7000,id:645,api:1,p:27887,c:466) queueBuffer: fps=11.49 dur=1566.67 max=1286.76 min=9.83
02-11 13:02:20.596 27887 28067 I e.companyporta: The ClassLoaderContext is a special shared library.
02-11 13:02:20.599 27887 28067 I System.out: [OkHttp] sendRequest<<
02-11 13:02:22.485 27887 27993 I EventHandler: Draining All events
02-11 13:02:22.486 27887 27993 I AbstractHandler: File e61639a8-46c1-4e99-94be-8e802fa394ff.norm.cllevent is in use still
02-11 13:02:22.486 27887 27993 I AbstractHandler: File 825dbb13-a008-4b54-b508-3456e37a1bde.crit.cllevent is in use still
02-11 13:02:22.486 27887 27993 I EventQueueWriter: Starting upload
02-11 13:02:22.487 27887 27993 I FileStorage: Read 1 events from file
02-11 13:02:22.495 27887 27993 I e.companyporta: The ClassLoaderContext is a special shared library.
02-11 13:02:22.497 27887 27993 I System.out: [OkHttp] sendRequest<<
02-11 13:02:22.564 27887 27895 W System  : A resource failed to call end.
02-11 13:02:22.570 27887 27895 I chatty  : uid=10109(com.microsoft.windowsintune.companyportal) FinalizerDaemon identical 7 lines
02-11 13:02:22.571 27887 27895 W System  : A resource failed to call end.
02-11 13:02:22.663 27887 27993 I EventSender: {"ipv":false,"pvm":null,"rej":0,"bln":0,"acc":1,"efi":[]}
02-11 13:02:22.663 27887 27993 I FileStorage: Discarding file
02-11 13:02:22.665 27887 27993 I FileStorage: Read 1 events from file
02-11 13:02:22.672 27887 27993 I e.companyporta: The ClassLoaderContext is a special shared library.
02-11 13:02:22.674 27887 27993 I System.out: [OkHttp] sendRequest<<
02-11 13:02:22.775 27887 27993 I EventSender: {"ipv":false,"pvm":null,"rej":0,"bln":0,"acc":1,"efi":[]}
02-11 13:02:22.776 27887 27993 I FileStorage: Discarding file
02-11 13:02:22.777 27887 27993 I FileStorage: Read 1 events from file
02-11 13:02:22.785 27887 27993 I e.companyporta: The ClassLoaderContext is a special shared library.
02-11 13:02:22.787 27887 27993 I System.out: [OkHttp] sendRequest<<
02-11 13:02:22.887 27887 27993 I EventSender: {"ipv":false,"pvm":null,"rej":0,"bln":0,"acc":1,"efi":[]}
02-11 13:02:22.887 27887 27993 I FileStorage: Discarding file
02-11 13:02:22.889 27887 27993 I FileStorage: Read 1 events from file
02-11 13:02:22.899 27887 27993 I e.companyporta: The ClassLoaderContext is a special shared library.
02-11 13:02:22.902 27887 27993 I System.out: [OkHttp] sendRequest<<
02-11 13:02:22.992 27887 27993 I EventSender: {"ipv":false,"pvm":null,"rej":0,"bln":0,"acc":1,"efi":[]}
02-11 13:02:22.992 27887 27993 I FileStorage: Discarding file
02-11 13:02:22.993 27887 27993 I EventQueueWriter: Sent 124 events.

From Authenticator app
02-11 13:06:16.485   466   507 I BufferQueueProducer: [PopupWindow:2a04d21#0](this:0x75d82b9000,id:654,api:1,p:28584,c:466) queueBuffer: fps=9.91 dur=1614.88 max=1378.02 min=10.08
02-11 13:06:16.824 28584 28584 D OpenSSLLib: OpensslErr:Module:12(177:); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:168;Function:ASN1_get_object
02-11 13:06:16.853   466  1221 I BufferQueueProducer: [com.azure.authenticator/com.azure.authenticator.ui.MainActivity#0](this:0x75d82aa800,id:653,api:1,p:28584,c:466) queueBuffer: fps=9.52 dur=1050.14 max=123.94 min=93.86
02-11 13:06:17.132 28584 28643 D Surface : Surface::disconnect(this=0x7d1e593000,api=1)
02-11 13:06:17.132   466  2310 I BufferQueueProducer: [PopupWindow:2a04d21#0](this:0x75d82b9000,id:654,api:1,p:28584,c:466) disconnect(P): api 1
02-11 13:06:17.134 28584 28584 D View    : [Warning] assignParent to null: this = android.widget.PopupWindow$PopupDecorView{4f3fd2 V.E...... R.....ID 0,0-492,363}
02-11 13:06:18.438   466  2310 I BufferQueueProducer: [com.azure.authenticator/com.azure.authenticator.ui.MainActivity#0](this:0x75d82aa800,id:653,api:1,p:28584,c:466) queueBuffer: fps=1.26 dur=1584.63 max=1578.86 min=5.76
02-11 13:06:19.603   466   507 I BufferQueueProducer: [com.azure.authenticator/com.azure.authenticator.ui.MainActivity#0](this:0x75d82aa800,id:653,api:1,p:28584,c:466) queueBuffer: fps=14.59 dur=1165.06 max=898.76 min=14.69
02-11 13:06:20.310   459   482 I vendor.mediatek.hardware.power@2.0-impl: notifyAppState pack:com.azure.authenticator, act:com.azure.authenticator.ui.AadRemoteNgcRegistrationActivity, pid:28584, state:1
02-11 13:06:20.314 28584 28584 W ActivityThread: handleWindowVisibility: no activity for token android.os.BinderProxy@17882e1
02-11 13:06:20.336 28584 28584 D ViewRootImpl[AadRemoteNgcRegistrationActivity]: hardware acceleration = true , fakeHwAccelerated = false, sRendererDisabled = false, forceHwAccelerated = false, sSystemRendererDisabled = false
02-11 13:06:20.347 28584 28584 V PhoneWindow: DecorView setVisiblity: visibility = 4, Parent = null, this = DecorView@d98adb4[]
02-11 13:06:20.348 28584 28584 D ViewRootImpl[AadRemoteNgcRegistrationActivity]: hardware acceleration = true , fakeHwAccelerated = false, sRendererDisabled = false, forceHwAccelerated = false, sSystemRendererDisabled = false
02-11 13:06:20.350 28584 28584 V PhoneWindow: DecorView setVisiblity: visibility = 0, Parent = android.view.ViewRootImpl@46b2823, this = DecorView@d98adb4[AadRemoteNgcRegistrationActivity]
02-11 13:06:20.359 28584 28584 D Surface : Surface::allocateBuffers(this=0x7d094cb000)
02-11 13:06:20.359 28584 28643 D Surface : Surface::connect(this=0x7d094cb000,api=1)
02-11 13:06:20.359   466   507 I BufferQueueProducer: [com.azure.authenticator/com.azure.authenticator.ui.AadRemoteNgcRegistrationActivity#0](this:0x75d37df800,id:655,api:1,p:28584,c:466) connect(P): api=1 producer=(28584:com.azure.authenticator) producerControlledByApp=true
02-11 13:06:20.360 28584 28643 D mali_winsys: EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig, egl_winsys_surface **, EGLBoolean) returns 0x3000
02-11 13:06:20.367 28584 28643 D Surface : Surface::connect(this=0x7d094cc000,api=1)
02-11 13:06:20.367 28584 28584 D Surface : Surface::allocateBuffers(this=0x7d094cc000)
02-11 13:06:20.367   466  2310 I BufferQueueProducer: [com.azure.authenticator/com.azure.authenticator.ui.AadRemoteNgcRegistrationActivity#1](this:0x75d371e000,id:656,api:1,p:28584,c:466) connect(P): api=1 producer=(28584:com.azure.authenticator) producerControlledByApp=true
02-11 13:06:20.368 28584 28643 D mali_winsys: EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig, egl_winsys_surface **, EGLBoolean) returns 0x3000
02-11 13:06:21.390   466  2310 I BufferQueueProducer: [com.azure.authenticator/com.azure.authenticator.ui.AadRemoteNgcRegistrationActivity#0](this:0x75d37df800,id:655,api:1,p:28584,c:466) queueBuffer: fps=60.25 dur=1012.46 max=18.69 min=13.43
02-11 13:06:22.280 28584 28630 W Telemetry:  [2020-02-11 13:06:22 - {"thread_id":"1238","correlation_id":"60f9eab1-3cef-45c6-9e25-809511aa8aac"}] Telemetry is disabled because the Telemetry context or configuration is null Android 28
02-11 13:06:22.320 28584 28608 I e.authenticato: The ClassLoaderContext is a special shared library.
02-11 13:06:22.322 28584 28608 I System.out: [OkHttp] sendRequest>>
02-11 13:06:22.322 28584 28608 I System.out: [OkHttp] sendRequest<<
02-11 13:06:22.335 28584 28701 I System.out: port:443
02-11 13:06:22.391   466  2310 I BufferQueueProducer: [com.azure.authenticator/com.azure.authenticator.ui.AadRemoteNgcRegistrationActivity#0](this:0x75d37df800,id:655,api:1,p:28584,c:466) queueBuffer: fps=59.93 dur=1001.15 max=19.86 min=14.11
02-11 13:06:22.476 28584 28701 D OpenSSLLib: OpensslErr:Module:12(177:); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:168;Function:ASN1_get_object
02-11 13:06:22.477 28584 28701 D OpenSSLLib: OpensslErr:Module:12(177:); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:168;Function:ASN1_get_object
02-11 13:06:22.548 28584 28701 I System.out: [OkHttp] sendRequest>>
02-11 13:06:22.548 28584 28701 I System.out: [OkHttp] sendRequest<<
02-11 13:06:22.631 28584 28701 D OpenSSLLib: OpensslErr:Module:12(177:); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:168;Function:ASN1_get_object
02-11 13:06:22.641 28584 28630 I AcquireTokenRequest:verifyBrokerRedirectUri:  [2020-02-11 13:06:22 - {"thread_id":"1238","correlation_id":"60f9eab1-3cef-45c6-9e25-809511aa8aac"}] This is a broker redirectUri. Bypass the check. Android 28
02-11 13:06:22.644 28584 28701 D OpenSSLLib: OpensslErr:Module:12(177:); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:168;Function:ASN1_get_object
02-11 13:06:22.648 28584 28630 I AcquireTokenRequest:tryAcquireTokenSilentWithBroker:  [2020-02-11 13:06:22 - 60f9eab1-3cef-45c6-9e25-809511aa8aac] Either could not get tokens from local cache or is force refresh request, switch to Broker for auth, clear tokens from local cache for the user. ver:1.16.3 Android 28
02-11 13:06:22.650 28584 28630 I AcquireTokenWithBrokerRequest:logBrokerVersion:  [2020-02-11 13:06:22 - 60f9eab1-3cef-45c6-9e25-809511aa8aac] Broker app is: com.microsoft.windowsintune.companyportal;Broker app version: VersionName=5.0.4689.0;VersonCode=3873113. ver:1.16.3 Android 28
02-11 13:06:22.663 28584 28630 E BrokerAccountServiceHandler:bindToBrokerAccountService:  [2020-02-11 13:06:22 - 60f9eab1-3cef-45c6-9e25-809511aa8aac] BROKER_BIND_SERVICE_FAILED:Failed to bind service to broker app.  ver:1.16.3 Android 28
02-11 13:06:22.665 28584 28630 E BrokerAccountServiceHandler:bindToBrokerAccountService:  [2020-02-11 13:06:22 - 60f9eab1-3cef-45c6-9e25-809511aa8aac] BROKER_BIND_SERVICE_FAILED:'bindService returned false ver:1.16.3 Android 28
02-11 13:06:22.667 28584 28630 E BrokerAccountServiceHandler:getAuthToken:  [2020-02-11 13:06:22 - 60f9eab1-3cef-45c6-9e25-809511aa8aac] BROKER_AUTHENTICATOR_NOT_RESPONDING:Get error when trying to bind the broker account service. ver:1.16.3 Android 28
02-11 13:06:22.668 28584 28630 E BrokerAccountServiceHandler:getAuthToken:  [2020-02-11 13:06:22 - 60f9eab1-3cef-45c6-9e25-809511aa8aac] BROKER_AUTHENTICATOR_NOT_RESPONDING:Failed to bind the service in broker app ver:1.16.3 Android 28
02-11 13:06:22.668 28584 28630 E BrokerAccountServiceHandler:getAuthToken: com.microsoft.aad.adal.AuthenticationException: Failed to bind the service in broker app
02-11 13:06:22.668 28584 28630 E BrokerAccountServiceHandler:getAuthToken: 	at com.microsoft.aad.adal.BrokerAccountServiceHandler.bindToBrokerAccountService(BrokerAccountServiceHandler.java:370)
02-11 13:06:22.668 28584 28630 E BrokerAccountServiceHandler:getAuthToken: 	at com.microsoft.aad.adal.BrokerAccountServiceHandler.performAsyncCallOnBound(BrokerAccountServiceHandler.java:326)
02-11 13:06:22.668 28584 28630 E BrokerAccountServiceHandler:getAuthToken: 	at com.microsoft.aad.adal.BrokerAccountServiceHandler.getAuthToken(BrokerAccountServiceHandler.java:132)
02-11 13:06:22.668 28584 28630 E BrokerAccountServiceHandler:getAuthToken: 	at com.microsoft.aad.adal.BrokerProxy.getAuthTokenInBackground(BrokerProxy.java:323)
02-11 13:06:22.668 28584 28630 E BrokerAccountServiceHandler:getAuthToken: 	at com.microsoft.aad.adal.AcquireTokenWithBrokerRequest.acquireTokenWithBrokerSilent(AcquireTokenWithBrokerRequest.java:69)
02-11 13:06:22.668 28584 28630 E BrokerAccountServiceHandler:getAuthToken: 	at com.microsoft.aad.adal.AcquireTokenRequest.tryAcquireTokenSilentWithBroker(AcquireTokenRequest.java:519)
02-11 13:06:22.668 28584 28630 E BrokerAccountServiceHandler:getAuthToken: 	at com.microsoft.aad.adal.AcquireTokenRequest.acquireTokenSilentFlow(AcquireTokenRequest.java:469)
02-11 13:06:22.668 28584 28630 E BrokerAccountServiceHandler:getAuthToken: 	at com.microsoft.aad.adal.AcquireTokenRequest.tryAcquireTokenSilent(AcquireTokenRequest.java:377)
02-11 13:06:22.668 28584 28630 E BrokerAccountServiceHandler:getAuthToken: 	at com.microsoft.aad.adal.AcquireTokenRequest.performAcquireTokenRequest(AcquireTokenRequest.java:356)
02-11 13:06:22.668 28584 28630 E BrokerAccountServiceHandler:getAuthToken: 	at com.microsoft.aad.adal.AcquireTokenRequest.access$200(AcquireTokenRequest.java:55)
02-11 13:06:22.668 28584 28630 E BrokerAccountServiceHandler:getAuthToken: 	at com.microsoft.aad.adal.AcquireTokenRequest$1.run(AcquireTokenRequest.java:129)
02-11 13:06:22.668 28584 28630 E BrokerAccountServiceHandler:getAuthToken: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
02-11 13:06:22.668 28584 28630 E BrokerAccountServiceHandler:getAuthToken: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
02-11 13:06:22.668 28584 28630 E BrokerAccountServiceHandler:getAuthToken: 	at java.lang.Thread.run(Thread.java:764)
02-11 13:06:22.690 28584 28643 D Surface : Surface::disconnect(this=0x7d094cb000,api=1)
02-11 13:06:22.690   466  1221 I BufferQueueProducer: [com.azure.authenticator/com.azure.authenticator.ui.AadRemoteNgcRegistrationActivity#0](this:0x75d37df800,id:655,api:1,p:28584,c:466) disconnect(P): api 1
02-11 13:06:22.691 28584 28584 D View    : [Warning] assignParent to null: this = DecorView@9fbc12d[AadRemoteNgcRegistrationActivity]
02-11 13:06:22.703 28584 28584 D ViewRootImpl[Toast]: hardware acceleration = true , fakeHwAccelerated = false, sRendererDisabled = false, forceHwAccelerated = false, sSystemRendererDisabled = false
02-11 13:06:22.711 28584 28584 D Surface : Surface::allocateBuffers(this=0x7d094cb000)
02-11 13:06:22.711 28584 28643 D Surface : Surface::connect(this=0x7d094cb000,api=1)
02-11 13:06:22.711   466  1221 I BufferQueueProducer: [Toast#0](this:0x75d06f5000,id:657,api:1,p:28584,c:466) connect(P): api=1 producer=(28584:com.azure.authenticator) producerControlledByApp=true
02-11 13:06:22.712 28584 28643 D mali_winsys: EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig, egl_winsys_surface **, EGLBoolean) returns 0x3000
02-11 13:06:22.716   459   482 I vendor.mediatek.hardware.power@2.0-impl: notifyAppState pack:com.azure.authenticator, act:com.azure.authenticator.ui.MainActivity, pid:28584, state:1
02-11 13:06:22.741 28584 28584 V PhoneWindow: DecorView setVisiblity: visibility = 0, Parent = android.view.ViewRootImpl@50c4a31, this = DecorView@324fe4a[MainActivity]
02-11 13:06:22.752   466  1221 I BufferQueueProducer: [com.azure.authenticator/com.azure.authenticator.ui.MainActivity#0](this:0x75d82aa800,id:653,api:1,p:28584,c:466) queueBuffer: fps=14.93 dur=3148.68 max=2366.32 min=3.71
02-11 13:06:22.758 28584 28584 V PhoneWindow: DecorView setVisiblity: visibility = 4, Parent = android.view.ViewRootImpl@46b2823, this = DecorView@d98adb4[AadRemoteNgcRegistrationActivity]
02-11 13:06:22.765 28584 28643 D Surface : Surface::disconnect(this=0x7d094cc000,api=1)
02-11 13:06:22.765   466   507 I BufferQueueProducer: [com.azure.authenticator/com.azure.authenticator.ui.AadRemoteNgcRegistrationActivity#1](this:0x75d371e000,id:656,api:1,p:28584,c:466) disconnect(P): api 1
02-11 13:06:22.766 28584 28584 D View    : [Warning] assignParent to null: this = DecorView@d98adb4[AadRemoteNgcRegistrationActivity]

**Comment by rpdome** [comment]
Hi,

There is a known issue regarding Bind Service on some Android OEM. The work around is to turn off power optimization in Authenticator and Company Portal's settings.

Thanks,
Dome

**Comment by malwaremagpie** [comment]
Resetting the device, installing the applications and setting their Battery Optimization to "off" still produced the same issue.  However, I found the fix!

I noticed this in the logs
`03-12 12:16:19.725  2138  6752 V Appblock: 应用限制--阻止关联启动, caller: com.azure.authenticator, receiver: com.microsoft.windowsintune.companyportal, Component: ComponentInfo{com.microsoft.windowsintune.companyportal/com.microsoft.aad.adal.BrokerAccountService}`

Appblock comes from the "App blocker" under "Intelligent assistance" in the settings menu.  It is on by default and has options to disable automatic of startup, launching of an app from another app, and background running of apps.  Turning that off for "Company Portal" and "Authenticator" did the job.
